tests/receivers/kong/testdata/server/kong.yml

```
@@ -0,0 +1,13 @@
+_format_version: "3.0"
+_transform: true
+
+services:
+  - host: httpbin.org
```

no need to refer to an external resource (don't think it's even used?)

```suggestion
  - host: localhost
```